src/client/updater.py

```diff
@@ -172,11 +182,34 @@ def server_session(self):
 
     def _parse_releases(self, release_info):
         def _parse_release(release_dict):
+            cv = Version(config.VERSION)
             for asset in release_dict['assets']:
                 if '.msi' in asset['browser_download_url']:
                     download_url = asset['browser_download_url']
                     tag = release_dict['tag_name']
+                    v = Version(tag)
+                    # We never want to return the current version itself,
+                    # but if `updater_downgrade` is set, we do return
+                    # older releases.
+                    # strange comparison logic is because of semantic_version
+                    # so that build info is ignored
+                    if self.updater_downgrade:
+                        if not v < cv or cv < v
```

Comment:
what screwed us here is that I left off the ':'. The precedence [is fine](https://docs.python.org/3/reference/expressions.html#operator-precedence).